front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -162,7 +191,13 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
           }}
         />
       </div>
-      <div className="stdcm-consist__properties">
+      <div
+        className={classNames({
+          'stdcm-consist__properties': !consistErrors.totalMass && !consistErrors.totalLength,
+          'stdcm-consist_with_errors__properties':
```

Comment:
I'm not familiar with stdcm: what is this `__properties` suffix? Is it a pattern (and what is the reason for this pattern)? Or is it required by something?

Reply:
I think it's a CSS naming convention called BEM.

Reply:
yes, this new class do not follow the convention, i will rename it to "consist__properties--with-errors"